tests/fixtures/on_demand_cluster_fixtures.py

```diff
@@ -66,6 +79,25 @@ def ondemand_aws_docker_cluster(request):
     return cluster
 
 
+@pytest.fixture(scope="session")
+def den_launched_ondemand_aws_docker_cluster(request):
+    """
+    Note: Also used to test docker and default env with alternate Ray version.
+    """
+    args = {
+        "name": "aws-cpu",
+        "instance_type": "CPU:2+",
+        "provider": "aws",
+        "image_id": "docker:rayproject/ray:latest-py311-cpu",
```

Comment:
isn't this failing when running locally? we don't have image / image_id wired up yet to send to Den for launching

Reply:
actually works pretty well! :)
attached is an output of `pytest --level release tests -k "den_launched_ondemand_aws_docker_cluster" --detached` (failures are not related to the cluster init)

![image](https://github.com/user-attachments/assets/8e3383f5-ab3e-4eeb-860a-c51d020d559f)